spec/fixtures/managed_modules.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Can you clarify?  This is a fixture, and not the actual file in the root of the repo.  This is necessary for testing.

Reply:
Hello @logicminds ,

We're using a before block in order to avoid editing that file and all the confusion about managed_modules file.
```
before(:all) do
    @pdksync_dir = './modules_pdksync'
    module_name = 'puppetlabs-testing'
    @module_names = ['puppetlabs-testing']
    @output_path = "#{@pdksync_dir}/#{module_name}"
    @folder = Dir.pwd
  end
```
Cheers!

Reply:
I renamed my fixtures file to avoid confusion.